### DISCUSSION TITLE
IR-Hub-Problems with  http:command

### DISCUSSION THREAD

**Discussion by CJS0815** [arendst/Tasmota#10220] [Q&A] [answered]
Hi All,

I have a Smart Wifi IR Hub flashed with tasmota-ir.bin (v9.2.0 and versions before).
At the console the command 
     IRsend {"Protocol":"NEC","Bits":"32","Data":"0x400501FE"}
was confirmed with
     CMD: IRsend {"Protocol":"NEC","Bits":"32","Data":"0x400501FE"}
and following
     RSL: stat/":"cmnd/tasmotas/"}/RESULT = {"IRSend":"Done"}
and everything works as expected

This command I need as a result from a shelly-plug s 
So I put the command as a action  by switch on the shelly plug s 
     http://192.168.1.100/cm?cmnd=irsend {"Protocol":"NEC","Bits":"32","Data":"0x400501FE"}
In response to the execute command the IR-Hub confirmed with
     RSL: stat/":"cmnd/tasmotas/"}/RESULT = {"IRSend":"No Bits or Data"}

Sorry I'm a absolute beginner. Can someone give me a hint what I'm doing wrong; in general parlance - as you read, my english is googles english ;-)

Grateful for all meaningful answers

Claus

**Comment by sfromis** [comment, marked as answer]
That messages is what you get from IRSend without any argument. I don't know the technicalities of how that it works with commands in that action, but you may need to escape characters, at least replacing the space after irsend by %20 or a plus sign +
https://tasmota.github.io/docs/Commands/#with-web-requests
Possibly, you also need to escape characters like {":}

**Reply by CJS0815** [reply]
Thanks a lot!

This shit "nothing" I've overlooked - shame on me ....

No it works fine as expected.
